src/main/java/duke/exceptions/InvalidKeyException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exceptions;
+
+/**
+ * InvalidKeyException is an exception that indicates that the key word is invalid.
+ */
+public class InvalidKeyException extends Exception{
+    /**
+     * Create a new InvalidKeyException.
```

Comment:
Should the starting word be `creates` instead of `create`? Same for the other Exception classes